contracts/extensions/VotingReputation.sol

```diff
@@ -0,0 +1,119 @@
+/*
+  This file is part of The Colony Network.
+
+  The Colony Network is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  The Colony Network is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with The Colony Network. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+pragma solidity 0.5.8;
+pragma experimental ABIEncoderV2;
+
+import "../PatriciaTree/PatriciaTreeProofs.sol";
+import "./VotingBase.sol";
+
+
+contract VotingReputation is VotingBase, PatriciaTreeProofs {
+
+  constructor(address _colony) public VotingBase(_colony) {}
+
+  struct RepInfo {
+    bytes32 rootHash;
+    uint256 skillId;
+  }
+
+  mapping (uint256 => RepInfo) repInfo;
+
+  // The UserVote type here is just the bytes32 voteSecret
+
+  mapping (address => mapping (uint256 => bytes32)) userVotes;
+
+  function setPollRepInfo(uint256 _pollId, uint256 _skillId) public onlyCreator(_pollId) pollPending(_pollId) {
+    repInfo[pollCount] = RepInfo({
```

Comment:
You're right, it should definitely use `_pollId`. Overriding is an option, but I didn't do it this time because

a) It would have technically been an overloading since the function signature would be different, which would have necessitated a second overriding (to throw an exception), and 

b) I figured why not let the poll creator update the reputation information arbitrarily up until the poll opens.

Although I could definitely do it the other way.

Reply:
Note that the shift to binary poll has allowed us to omit the entire `Pending` phase and thus this functionality.